sebs/cloudflare/cloudflare.py

```
@@ -0,0 +1,1608 @@
+            docker_client,
+            CloudflareSystemResources(config, cache_client, docker_client, logger_handlers),
+        )
+        self.logging_handlers = logger_handlers
+        self._config = config
+        self._api_base_url = "https://api.cloudflare.com/client/v4"
+        # cached workers.dev subdomain for the account (e.g. 'marcin-copik')
+        # This is different from the account ID and is required to build
+        # public worker URLs like <name>.<subdomain>.workers.dev
+        self._workers_dev_subdomain: Optional[str] = None
+
+    def initialize(self, config: Dict[str, str] = {}, resource_prefix: Optional[str] = None):
```

Ruff (0.14.10)

Unused method argument: `config`

(ARG002)